src/lib/util/Games/Jackpot.js

```diff
@@ -0,0 +1,40 @@
+class Jackpot {
+
+	constructor(client) {
+		this.client = client;
+		this.users = [];
+	}
+
+	async add(id, amount) {
+		this.users.push({ id, amount });
+		if (await this.client.providers.default.has('jackpot', id)) await this.client.providers.default.update('jackpot', id, { amount });
+		else await this.client.providers.default.create('jackpot', id, { amount });
+		return this.users.find(jackpotUser => jackpotUser.id === id).amount;
+	}
+
+	async draw() {
+		if (this.users.length === 0) return null;
```

Comment:
```suggestion
		if (this.users.length === 0) return { id: null, amount: 0 };
```

And make the task always deconstruct draw's output, just check if id is null.